app/src/main/java/ar/com/wolox/android/example/ui/home/News/NewsAdapter.kt

```diff
@@ -0,0 +1,64 @@
+package ar.com.wolox.android.example.ui.home.News
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.model.News
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+
+class NewsAdapter : RecyclerView.Adapter<NewsAdapter.ViewHolder>() {
+
+    private var listOfNews = arrayListOf<News>()
+    private lateinit var view: View
+    private lateinit var context: Context
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        context = parent.context
+        view = LayoutInflater.from(parent.context).inflate(ar.com.wolox.android.R.layout.news_item, parent, false)
+        return ViewHolder(view)
+    }
+
+    fun setNews(listOfNews: List<News>) {
+        this.listOfNews.clear()
+        this.listOfNews.addAll(listOfNews)
+    }
+
+    fun clear() {
+        this.listOfNews.clear()
+    }
+
+    fun addNews(listOfNews: List<News>) {
+        this.listOfNews.addAll(listOfNews)
+    }
+
+    override fun getItemCount(): Int {
+        return listOfNews.size
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val news: News = listOfNews[position]
+        holder.author.text = news.title
```

Comment:
I think that you can use holder.apply {  <Your code> }. This allows you to avoid writing **holder. autor.text** and you only use **autor.text**